rdr_service/model/participant_summary.py

```diff
@@ -1555,7 +1555,8 @@ def enrollmentSiteId(cls):
     """User who recorded ID verification occurrence"""
     onsiteIdVerificationSite = Column("onsite_id_verification_site_id", Integer, ForeignKey("site.site_id"))
     """The site where ID verification took place"""
-
+    aian = Column(Boolean, default=0)
+    """Denotes if the participants race is AI_AN (AMERICAN_INDIAN_OR_ALASKA_NATIVE)"""
```

Comment:
Would it be better to leave this defaulting as Null (which would show up as "UNSET" in the API). That way it doesn't look like we have a definite answer for whether a participant is AIAN when we haven't gotten their TheBasics response yet (we'd show "UNSET" until we get a response, and then we would show Yes or No). I'm realizing now that this would also need a change in how this field gets set when we get a response.

Reply:
Understood.